hummingbird/ml/operator_converters/_imputer_implementations.py

```diff
@@ -0,0 +1,81 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Base classes for Imputers
+"""
+
+import torch
+import numpy as np
+
+from ._physical_operator import PhysicalOperator
+from . import constants
+
+
+class SimpleImputer(PhysicalOperator, torch.nn.Module):
+    """
+    Class implementing SimpleImputer operators in PyTorch.
+    """
+
+    def __init__(self, logical_operator, device, statistics=None, missing=None, strategy=None):
+        super(SimpleImputer, self).__init__(logical_operator)
+        sklearn_imputer = logical_operator.raw_operator
+        # Pull out the stats field from either the SKL imputer or args
+        stats_ = statistics if statistics is not None else sklearn_imputer.statistics_
+        # Process the stats into an array
+        stats = [float(stat) for stat in stats_ if isinstance(stat, float)]
```

Comment:
Why do you need to check the type here? I think we can skip this and just use straight stats?

Reply:
(I know that this is not your code)